config/02-ce-sge.conf

```diff
@@ -7,6 +7,13 @@
 ###############################################################################
 
 # Basic route for submitting to SGE
+JOB_ROUTER_ROUTE_SGE_Cluster @=jrt
+  TargetUniverse = 9
+  GridResource = "batch sge"
+@jrt
+
+JOB_ROUTER_ROUTE_NAMES = SGE_Cluster
+
```

Comment:
Should we remove the deprecation warning from this config file? Willing to keep in given the commented out old syntax configuration